### ISSUE TITLE
running wmllint against ANL

### ISSUE THREAD

**Issue by sevu** [wesnoth/wesnoth#4382]
Running wmllint against ANL discovers:
```
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 448: side number 5 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 452: side number 5 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 476: side number 6 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 480: side number 6 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 504: side number 7 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 508: side number 7 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 532: side number 8 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 536: side number 8 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 769: side number 1 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 782: side number 2 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 795: side number 3 is out of sequence (11 expected)
"data/multiplayer/scenarios/4p_A_New_Land.cfg", line 808: side number 4 is out of sequence (11 expected)
```
This points to `side=` in tags like [store_side] or [modify_side] inside [event]s.
I suspect this is a bug in wmllint.

Also still happening if one deletes all the preprocessor directives (macro-usage, defines, ifdefs)

**Comment by gfgtdf** [comment]
honestly, i think we should move the ANL code to lua anyways if we find time to do that.

**Comment by Elvish-Hunter** [comment]
I'm closing this issue because it was fixed in 038f0d5.
